backend/inspirehep/migrator/api.py

```diff
@@ -39,15 +39,22 @@ def continuous_migration():
         try:
             migrated_records = None
             num_of_records = r.llen("legacy_records")
-            LOGGER.info(f"Starting migration of {num_of_records} records.")
+            LOGGER.info(
+                f"Starting migration of {num_of_records} records.",
```

Comment:
Isn't the point of structured logging to have the vars outside of the message, which serves only for the type of message? so
```suggestion
                "Starting migration of records.",
```